iterations_runner.lua

```diff
@@ -40,6 +40,18 @@ local ffi = require("ffi")
 local json = require("json_nojit")
 local jitstats
 
+local devnull
+if package.config:sub(1,1) == "\\" then
+  -- Use nul on windows https://stackoverflow.com/questions/313111/is-there-a-dev-null-on-windows
```

Comment:
Does that mean that one can never have a file called `nul` on Windows? That seems weird!

Reply:
Its considered a reserve name so its not something you can create though any normal means.

Reply:
OK